src/box/sql/func.c

```diff
@@ -2374,6 +2374,10 @@ func_sql_expr_call(struct func *func, struct port *args, struct port *ret)
 	struct vdbe_field_ref *ref;
 	size_t size = sizeof(ref->slots[0]) * count + sizeof(*ref);
 	ref = region_aligned_alloc(region, size, alignof(*ref));
+	if (ref == NULL) {
+		diag_set(OutOfMemory, size, "region_aligned_alloc", "ref");
+		goto error;
+	}
```

Comment:
Please use `xregion_aligned_alloc`. I'd prefer if you prepared one patch (in a separate PR) that replaces all region allocations in the SQL code with `x*` versions in one go rather than fixing each place individually.

Reply:
Do we actually have `xregion_aligned_alloc()` Should I add this function?